src/apps/activities/tests/test_reusable_items.py

```diff
@@ -24,7 +24,7 @@ async def test_create_item_choice(self):
         response = await self.client.post(self.create_url, data=create_data)
 
         assert response.status_code == 201, response.json()
-        assert response.json()["Result"]["Id"] == 1
+        assert response.json()["result"]["id"] == 1
```

Comment:
We can't have the data without the status code :)
So there is no need to test it.
Also, I think if you test the endpoint that is creating something you also have to check the number of database records instead of comparing the response schema. Don't you think?